pages/news/asteroid-hosttools.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Introducing asteroid-hosttools
+layout: news
+posted: 2023-04-24
+post: true
+thumbnails: asteroid-hosttools
+summary: The asteroid-hosttools make it convenient to work with AsteroidOS smartwatches from a Linux computer.
+author: Timo Könnecke (eLtMosen)
+---
+# Backup and restore has never been easier
+
+With the new software called "[asteroid-hosttools](https://github.com/beroset/asteroid-hosttools)" written by Ed Beroset, it's now easier and faster to work with AsteroidOS smartwatches from a Linux computer. The software comprises three tools: "flashy", "watch-image" and "watch-util".
+
+## Flashy
+Deploys an AsteroidOS image to a supported watch connected via USB easily. Nightly builds are automatically download into the current directory and are used to set up a temporary installation or flash the watch permanently using the real installation.\
```

Comment:
Not sure if that's me not being a native speaker but I expected "automatically downloaded" rather than download ?

Reply:
Good catch. No, its me not having been thorough enough :P

Reply:
You're right it should be "downloaded".